src/wp-includes/class-wp-xmlrpc-server.php

```diff
@@ -335,6 +351,23 @@ public function escape( &$data ) {
 		}
 	}
 
+	/**
+	 * @param IXR_Error|string $error
+	 * @param false $message
+	 */
```

Comment:
```suggestion
	/**
	 * Send error response to client.
	 *
	 * Send an XML error response to the client. If the endpoint is enabled
	 * an HTTP 200 response is always sent per the XML-RPC specification.
	 *
	 * @since 5.7.3
	 *
	 * @param IXR_Error|string $error   Error code or an error object.
	 * @param false            $message Error message. Optional.
	 */
```